### ISSUE TITLE
Querying if given range allows for an event

### ISSUE THREAD

**Issue by arshaw** [fullcalendar/fullcalendar#2890]
A method, given a date range, that returns if the given timespan is all-okay for adding a new event. Computed based off of `eventOverlap` and `eventConstraint`.

Suggested by `eyal`

**Comment by BudgieInWA** [comment]
:+1: I need this to properly implement a solution to my "[dropping events via click](http://stackoverflow.com/questions/32494784/in-fullcallendar-can-i-get-behaviour-similar-to-droppable-without-using-dragging)" problem.

**Comment by BudgieInWA** [comment]
Wait, I just discovered the exported 'isExternalDropRangeAllowed' method and its siblings. This is exactly what I need for my previously mentioned use case. Does this issue describe a different feature, or could it simply be resolved by describing these methods in the docs?

**Comment by arshaw** [comment]
yes, isExternalDropRangeAllowed would probably work for you, but before making it a public API method, I'd want to add further checks to the input, and possible rename it.

**Comment by arshaw** [comment]
I'm to consider this issue a duplicate of #2740 and close it. I will fill out that ticket with more info. Can you please subscribe to it? It should be applicable to your situation as well. Stay tuned...

**Comment by arshaw** [comment]
Reopening this, as I'm realizing it's its own distinct features. I can see the API for things being something like this:
```
var isAllowed = calendarObject.isEventRangeAllowed(eventStart, eventEnd);
```

**Comment by arshaw** [comment]
similar to https://github.com/fullcalendar/fullcalendar/issues/2754
